src/selection.ts

```diff
@@ -525,14 +529,14 @@ export class DataSelection {
     }
     await Promise.all(preparation)
     */
-    const columns = Object.fromEntries(fields.map((field) => [field, []]));
-    for (let row of this) {
-      for (let field of fields) {
-        columns[field].push(row[field]);
-      }
-    }
-    return columns;
-  }
+  //   const columns = Object.fromEntries(fields.map((field) => [field, []]));
```

Comment:
This seems somewhat substantial? Is this desired?

Reply:
Oh yeah, this is stuff intended to support atlas-next that we don't actually use in this form.

Reply:
So it creates useless type errors